backend/api/Areas/Property/Models/Search/GeoJsonFilterModel.cs

```diff
@@ -74,6 +74,12 @@ public class GeoJsonFilterModel : PageFilter
         /// <value></value>
         public bool? IgnorePropertiesInProjects { get; set; }
 
+        /// <summary>
```

Comment:
Realized that this may be redundant, should I just send `PropertyType.Land` as the `PropertyType` filter instead of creating a new flag?

Reply:
I think it already has the `PropertyType` filter.  You should just use it instead of a duplicate filter that does less.